subiquity/client/client.py

```diff
@@ -120,13 +120,13 @@ def make_ui(self):
 
     variant_to_controllers: Dict[str, List[str]] = {}
 
-    def __init__(self, opts):
+    def __init__(self, opts, about_msg=None):
```

Comment:
This was what I originally wanted, but gave up in the middle of the process and you guys encouraged me doing 😄 
Thanks for that.